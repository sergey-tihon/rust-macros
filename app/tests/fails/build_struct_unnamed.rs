use builder_macro::Builder;

#[derive(Builder)]
pub struct Point(i32, i32);

fn main() {}
