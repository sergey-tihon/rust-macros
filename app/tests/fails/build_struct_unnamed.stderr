error: only structs with named fields are supported. Struct: Point
 --> tests/fails/build_struct_unnamed.rs:4:1
  |
4 | pub struct Point(i32, i32);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
